src/containers/NotificationsContainer/NotificationsContainer.vue

```diff
@@ -171,6 +171,12 @@ export default {
   watch: {
     notifications() {
       this.countUnread();
+    },
+    network() {
+      this.pruneNotifications();
+    },
+    ['account.address']() {
+      this.pruneNotifications();
```

Comment:
Prune whenever there is a new notification, not when it changes the network or address